### PR TITLE
feat: include nodes count in operator usage endpoint and cli command

### PR DESCRIPTION
**Pull request by JadhavPoonam**
### Description

Enhances the `operator/usage` API endpoint and the corresponding CLI command to include node count in the output.

### Testing & Reproduction steps
Tested in dev mode

API

Screenshot 2023-06-28 at 4 26 37 PM

CLI
Screenshot 2023-06-29 at 4 36 50 PM

### Links

API doc: https://developer.hashicorp.com/consul/api-docs/operator/usage

### PR Checklist

* [ ] ~updated test coverage~
* [x] external facing docs updated
* [x] appropriate backport labels added
* [ ] ~not a security concern~

### CODE REVIEW

**Review by loshz — APPROVED** [review]
Great work!

File: command/operator/usage/instances/usage_instances.go [hunk 1]
```diff
@@ -99,6 +99,14 @@ func (c *cmd) Run(args []string) int {
...
 		}
 		c.UI.Output(billableOutput + "\n")
+
+		c.UI.Output("\nNodes")
+		nodesOutput, err := formatNodesCounts(usage.Usage)
+		if err != nil {
+			c.UI.Error(err.Error())
+			return 1
+		}
+		c.UI.Output(nodesOutput + "\n\n")
```
Comment (dhiaayachi) [inline command/operator/usage/instances/usage_instances.go:109]: Is the `\n\n` here on purpose?

**Review by dhiaayachi — APPROVED** [review]
👏 Great work @JadhavPoonam! I left a small question but nothing blocking.

**Review by dhiaayachi — APPROVED** [review]
changelog ✅

**Review by im2nguyen — APPROVED** [review]

**Pull request merged**